ale/drivers/keys.py

```diff
@@ -0,0 +1,64 @@
+base = {
+    'name_model'
+    'center_ephemeris_time',
+    'detector_center',
+    'detector_line_summing',
+    'detector_sample_summing',
+    'dt_ephemeris',
+    'dt_quaternion',
+    'ending_ephemeris_time',
+    'exposure_duration',
+    'focal2pixel_lines',
+    'focal2pixel_samples',
+    'focal_epsilon',
+    'focal_length',
+    'focal_plane_tempature',
+    'id_lookup',
+    'ikid',
+    'image_lines',
+    'image_samples',
+    'instrument_id',
+    'interpolation_method',
+    'reference_frame',
+    'reference_height',
+    'semimajor',
+    'semiminor',
+    'sensor_orientation',
+    'sensor_position',
+    'sensor_velocity',
+    'spacecraft_clock_stop_count',
+    'spacecraft_id',
+    'spacecraft_name',
+    'start_time',
+    'starting_detector_line',
+    'starting_detector_sample',
+    'starting_ephemeris_time',
+    'sun_position',
+    'sun_velocity',
+    'target_name',
+    'number_of_ephemerides',
+    'fikid',
+    'number_of_ephemerides'
+}
+
+transverse_distortion = {
+    'odtk',
+    'odtx'
+}
+
+radial_distortion = {
+    'odty'
+}
+
+framer = {
+    'filter_number',
+}
+
+linescanner = {
+    'line_exposure_duration',
+    'line_scan_rate',
+    't0_ephemeris',
+    't0_quaternion',
+    'dt_ephemeris',
+    'dt_quaternion',
+}
```

Comment:
It's weird if there is a duplication of keys between lists, but should not cause an issue.

If implementation classes implement the same key, then there can be conflicts based on order of inheritance in base classes.